src/main/java/core/basesyntax/FigureDataInterface.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface FigureDataInterface {
```

Comment:
Don't use verbs for class/interface names. In this case, 'FigureDataInterface' could be better named to reflect it's a property of figures, such as 'FigureData'.